docs/networking/port-forwarding.md

```diff
@@ -21,6 +22,16 @@ coder port-forward myworkspace --tcp 8000:8080
 
 For more examples, see `coder port-forward --help`.
 
+## Dashboard
+
+> To enable port forwarding via the dashboard, Coder must be configured with a
+> [wildcard access URL](./admin/configure#wildcard-access-url).
+
+Use the "Port forward" button in the dashboard to access ports
+running on your workspace.
+
+![Port forwarding in the UI](../images/port-forward-dashboard.png)
```

Comment:
Screenshot should read `port, agent name, workspace name and your username` (agent name before workspace name)